app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoStore.kt

```diff
@@ -0,0 +1,91 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.pocket
+
+import android.content.Context
+import android.content.res.AssetManager
+import android.util.Log
+import androidx.annotation.AnyThread
+import org.mozilla.tv.firefox.telemetry.SentryIntegration
+
+private const val LOGTAG = "PocketVideoStore"
+
+private const val KEY_VIDEO_JSON = "video_json"
+private const val VIDEO_STORE_NAME = "Pocket-Global-Video-Recs"
+
+private const val BUNDLED_VIDEOS_PATH = "bundled/pocket_videos.json"
+
+/**
+ * The minimum number of valid videos we must receive from the server if we want to display them.
+ * This number is set to the number of videos that appear on screen at the time of writing.
+ */
+private const val REQUIRED_POCKET_VIDEO_COUNT = 4
+
+/**
+ * Saves the Pocket video recommendations as a raw JSON String and loads them in data structures for the app.
+ * Bad data should saved so bad data should never be returned.
```

Comment:
`Bad data should saved so bad data should never be returned` 

Should this say `Bad data should not be saved`?

Reply:
Yes